lib/xcpretty/parser.rb

```diff
@@ -103,7 +103,7 @@ module Matchers
     # $2 = test_suite
     # $3 = test_case
     # $4 = reason
-    FAILING_TEST_MATCHER = /^\s*(.+:\d+):\serror:\s[\+\-]\[(.*)\s(.*)\]\s:(?:\s'.*'\s\[FAILED\],)?\s(.*)/
+    FAILING_TEST_MATCHER = /^\s*(.+?:\d+):\serror:\s[\+\-]\[([^\s]*)\s([^\]]*)\]\s:(?:\s'.*'\s\[FAILED\],)?\s(.*)/
```

Comment:
Instead of `.+?` you can do `[^:]+` which will save some more steps.

Reply:
yes, but it fails the tests